### PR TITLE
Task/g352 aggregate bonded announcing

### PR DESCRIPTION
**Pull request by rg911**
Added service methods in TransactionService for aggregate bonded transaction announcing

Issue: #352 

**Rebased from #363**

### CODE REVIEW

**Comment by dgarcia360** [comment]
Since the PR includes #363 logic, I would wait until the previous PR is merged to review fewer lines of code.

File: test/model/receipt/Statement.spec.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    it('should get reolved mosaic from receipt', () => {
+        const unresolvedMosaic = UnresolvedMapping.toUnresolvedMosaic('E81F622A5B11A340');
+        const statement = CreateStatementFromDTO(statementDTO, NetworkType.MIJIN_TEST);
+        const resolved = statement.resolveMosaicId(unresolvedMosaic as NamespaceId, '1473', 0);
+
+        expect(resolved instanceof MosaicId).to.be.true;
+        expect((resolved as MosaicId).equals(new MosaicId('756482FB80FD406C'))).to.be.true;
+    });
+
+    it('should get reolved mosaic from receipt without Harvesting_Fee', () => {
```
Comment (evias) [inline test/model/receipt/Statement.spec.ts]: typo resolved

File: test/model/receipt/Statement.spec.ts [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        };
+        const unresolvedAddress = UnresolvedMapping.toUnresolvedAddress('9156258DE356F030A500000000000000000000000000000000');
+        const statement = CreateStatementFromDTO(statementWithoutHarvesting, NetworkType.MIJIN_TEST);
+        const resolved = statement.resolveAddress(unresolvedAddress as NamespaceId, '1473', 0);
+
+        expect(resolved instanceof Address).to.be.true;
+        expect((resolved as Address).equals(account.address)).to.be.true;
+    });
+
+    it('should get reolved mosaic from receipt', () => {
```
Comment (evias) [inline test/model/receipt/Statement.spec.ts]: typo resolved

File: test/model/receipt/Statement.spec.ts [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    it('should get reolved address from receipt', () => {
+        const unresolvedAddress = UnresolvedMapping.toUnresolvedAddress('9156258DE356F030A500000000000000000000000000000000');
+        const statement = CreateStatementFromDTO(statementDTO, NetworkType.MIJIN_TEST);
+        const resolved = statement.resolveAddress(unresolvedAddress as NamespaceId, '1473', 0);
+
+        expect(resolved instanceof Address).to.be.true;
+        expect((resolved as Address).equals(account.address)).to.be.true;
+    });
+
+    it('should get reolved address from receipt without Harvesting_Fee', () => {
```
Comment (evias) [inline test/model/receipt/Statement.spec.ts]: typo resolved

File: test/model/receipt/Statement.spec.ts [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+        ];
+
+        statementDTO = {
+            transactionStatements: transactionStatementsDTO,
+            addressResolutionStatements: addressResolutionStatementsDTO,
+            mosaicResolutionStatements: mosaicResolutionStatementsDTO,
+        };
+    });
+
+    it('should get reolved address from receipt', () => {
```
Comment (evias) [inline test/model/receipt/Statement.spec.ts]: typo resolved

**Pull request merged**